include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_reduce.h

```diff
@@ -0,0 +1,383 @@
+// -*- C++ -*-
+//===-- parallel_backend_sycl_radix_sort.h --------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ONEDPL_PARALLEL_BACKEND_SYCL_REDUCE_H
+#define _ONEDPL_PARALLEL_BACKEND_SYCL_REDUCE_H
+
+#include "sycl_defs.h"
+#include "parallel_backend_sycl_utils.h"
+#include "execution_sycl_defs.h"
+#include "unseq_backend_sycl.h"
+#include "utils_ranges_sycl.h"
+
+namespace oneapi
+{
+namespace dpl
+{
+namespace __par_backend_hetero
+{
+
+template <typename... _Name>
+class __reduce_seq_kernel;
+
+template <::std::uint16_t /* __work_grou_size */, ::std::uint16_t /* __iters_per_work_item */, typename... _Name>
```

Comment:
Instead of having the meaning of the template parameters in comments, could you make them the names? Such as:
```c++
template <::std::uint16_t _WorkGroupSize, ::std::uint16_t _ItersPerWorkItem, typename... _Name>
```

Reply:
I will change that.